### DISCUSSION TITLE
Symbols and Selections for Mensa

### DISCUSSION THREAD

**Discussion by Nicole4Fox** [astarub/campus_app#128] [General]
- Currently the Symbols are not explained, there should be information about what the symbols mean in the same are that explains the short terms and additional information about food
- There should be an option to only show meat and fish for people who do a low carb diet besides vegetarian, vegan and halal
- Currently there is no explaination what dishes are deemed vegetarian/vegan. For example honey or eggs is not that clear. There should be additional information about what is classified about vegan/vegetarian/etc

**Comment by henry-herrmann** [comment]
1. For the symbols like Halal etc. I'll add the corresponding symbols to the preferences tab. Regarding the ingredients, all shortcodes are listed under the `Allergenes` tab. See image below.
2. That's doable, will put it on the todo list.
3. The actual definition of the dishes is done by the AKFÖ, on which we have no influence, so the shortcodes listed with each meal are the only indicators even we have. 

![image](https://github.com/astarub/campus_app/assets/44099069/331a735d-4856-449d-bd02-e9f73b51124f)
